### PR TITLE
JAT

### PR DESCRIPTION
**Pull request by qgallouedec**

### CODE REVIEW

File: jat.md [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+```
+
+In addition to RL data, we include textual datasets to enable a unique interface for the user. That's why you'll also find subsets for [Wikipedia](https://huggingface.co/datasets/wikipedia), [Oscar](https://huggingface.co/datasets/oscar), [OK-VQA](https://okvqa.allenai.org) and [Conceptual-Captions](https://huggingface.co/datasets/conceptual_captions).
+
+## JAT agent architecture
+
+JAT's architecture is based on a Transformer, using [EleutherAI's GPT-Neo implementation](https://huggingface.co/docs/transformers/model_doc/gpt_neo). JAT's particularity lies in its embedding mechanism, which has been built to intrinsically handle sequential decision tasks. We interleave observation embeddings with action embeddings, along with the corresponding rewards.
+
+<figure class="image text-center">
+  <img src="https://huggingface.co/datasets/huggingface/documentation-images/raw/main/blog/jat/model.svg" height="400" alt="Model">
```
Comment (ClementRomac) [inline jat.md:51]: I'm seeing broken images for observations. Is that normal?

File: jat.md [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+  <figcaption>Aggregated expert normalized scores with 95% Confidence Intervals (CIs) for each RL domain as a function of learning step.</figcaption>
```
Comment (ClementRomac) [inline jat.md:52]: The figure is showing the architecture, not the scores?

**Review by ClementRomac — COMMENTED** [review]
A couple of comments on the model's figure, otherwise looks great!

File: jat.md [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+- user: qgallouedec
+- user: edbeeching
+- user: ClementRomac
+---
+
+# Jack of All Trades, Master of Some, a Multi-Purpose Transformer Agent
+
+## Introduction
+
+We're excited to share Jack of All Trades (JAT), a project that aims to move in the direction of a generalist agent. In a nutshell, the project has resulted in:
```
Comment (thomwolf) [inline jat.md:14]:
```suggestion
We're excited to share Jack of All Trades (JAT), a project that aims to move in the direction of a generalist agent. The project started as an open reproduction of the Gato (Reed et al., 2022) work, which proposed to train a Transformer able to perform both vision-and-language and decision-making tasks. We thus started by building an open version of Gato’s dataset. We then trained multi-modal Transformer models on it, introducing several improvements over Gato for handling sequential data and continuous values.

Overall, the project has resulted in:
```

File: jat.md (hunk 1 above)
Comment (qgallouedec) [reply jat.md:51]: Strange. I noticed the same thing when you posted the message, but this morning it seems to have been fixed. Can you confirm?

**Review by ClementRomac — APPROVED** [review]
Looks great!

**Pull request merged**